lib/Sema/TypeCheckType.cpp

```diff
@@ -2221,8 +2221,21 @@ Type TypeResolver::resolveAttributedType(TypeAttributes &attrs,
 
     for (auto i : FunctionAttrs) {
       if (attrs.has(i)) {
-        TC.diagnose(attrs.getLoc(i), diag::attribute_requires_function_type,
-                    TypeAttributes::getAttrName(i));
+          if (attrs.has(TAK_escaping)) {
+              auto &SM = TC.Context.SourceMgr;
+              auto loc = attrs.getLoc(TAK_escaping);
```

Comment:
I think I would need to call `getRangeWithAt()` on an `attr`. How do I get the `attr` from `attrs`?